src/modules/private-share-folder/private-sharing.repository.ts

```diff
@@ -0,0 +1,50 @@
+import { Injectable } from '@nestjs/common';
+import { Folder } from '../folder/folder.domain';
+import { InjectModel } from '@nestjs/sequelize';
+import { PrivateSharingFolder } from './private-sharing-folder.domain';
+import { PrivateSharingFolderModel } from './private-sharing-folder.model';
+import { FolderModel } from '../folder/folder.model';
+
+export interface PrivateSharingRepository {
+  findSharedByMePrivateFolders(
+    userId: number,
+    offset: number,
+    limit: number,
+    orderBy?: [string, string][],
+  ): Promise<Folder[]>;
+}
+
+@Injectable()
+export class SequelizePrivateSharingRepository
+  implements PrivateSharingRepository
+{
+  constructor(
+    @InjectModel(PrivateSharingFolderModel)
+    private privateSharingFolderModel: typeof PrivateSharingFolderModel,
+    @InjectModel(FolderModel)
+    private folderModel: typeof FolderModel,
+  ) {}
+  async findSharedByMePrivateFolders(
```

Comment:
I find this naming too much-business related. I would use a naming more related to persistence like `findByOwner` or `findByOwnerWithFolders`